packages/oui-popover/src/popover.controller.js

```diff
@@ -119,6 +138,9 @@ export default class PopoverController {
 
         // Support for attribute `oui-popover`
         this.$element.attr("aria-expanded", true);
+
+        // force the digest because the popover is outside the angular digest loop
+        this.$timeout(() => this.onOpen(), 0);
```

Comment:
```suggestion
        this.$timeout(() => this.onOpen());
```

is enough, `0` is the default value for the second parameter :)